servers/lumen/app/Http/Controllers/WeightController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace App\Http\Controllers;
+
+use App\Weight;
+use Illuminate\Http\Request;
+use DB;
+use Predis\Autoloader;
+
+\Predis\Autoloader::register();
+
+class WeightController extends Controller
+{
+    public function getAllTemporalWeights()
+    {
+        $weightQuery = DB::select(DB::raw('SELECT *, round(UNIX_TIMESTAMP(ROW_START) * 1000) as ROW_START, round(UNIX_TIMESTAMP(ROW_END) * 1000) as ROW_END, UNIX_TIMESTAMP(timeOfMeasurementInMilliseconds) * 1000 as timeOfMeasurementInMilliseconds FROM sc_vital_signs.weight FOR SYSTEM_TIME ALL order by ROW_START desc'));
+
+        return response()->json($weightQuery);
+    }
+
+    public function getOneWeight($serverSideRowUuid)
+    {
+        $weightQuery = DB::select(DB::raw("SELECT *, round(UNIX_TIMESTAMP(ROW_START) * 1000) as ROW_START, round(UNIX_TIMESTAMP(ROW_END) * 1000) as ROW_END, UNIX_TIMESTAMP(timeOfMeasurementInMilliseconds) * 1000 as timeOfMeasurementInMilliseconds FROM sc_vital_signs.weight FOR SYSTEM_TIME ALL WHERE serverSideRowUuid LIKE '{$serverSideRowUuid}' order by ROW_START desc"));
+
+        return response()->json($weightQuery);
+    }
+
+
+    /*
+    To check weight/create in postman make a post request with the following JSON:
+    Open postman desktop app and then:
+    URL: http://localhost:8000/public/api/weight/v20/
+    Post / Body raw: {"rowToUpsert":{"$serverSideRowUuid":"3","vnRowStateInSession":34,"validationClass":"","isValidationError":false,"clientSideUniqRowId":3,"serverSideRowUuid":"01817fb0-c1ef-11ea-a3a5-f36fe4d74da4","weightInFarehnite":100,"timeOfMeasurementInMilliseconds":1601876131994,"notes":"test","recordChangedByUuid":"bfe041fa-073b-4223-8c69-0540ee678ff8","recordChangedFromIPAddress":"::1","recordChangedFromSection":"null","ROW_START":1601877241687,"ROW_END":2147483648000}}
+    */
+
+
+
+    public function create(Request $request)
+    {
+        $requestData = $request->all();
+
+        $serverSideRowUuid = $requestData['data']['serverSideRowUuid'];
+        $ptUuid = $requestData['data']['ptUuid'];
+        $timeOfMeasurementInMilliseconds = (int)($requestData['data']['timeOfMeasurementInMilliseconds']);
+        $weightInFarehnite = $requestData['data']['weightInFarehnite'];
```

Comment:
How to measure weight in 'Farehnite' scale?
Field name should be 'weightInPounds'.